File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        ans = []
        counter = {}
        for num in nums:
            if num in counter:
                counter[num] += 1
            else:
                counter[num] = 1
        
        arr = []
        for num, cnt in counter.items():
            arr.append([cnt,num])
        arr.sort(reverse = True)
        
        for i in range(k):
            ans.append(arr[i][1])

        return ans

File: Data Structures & Algorithms/is-anagram/submission-1.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if (s.length() != t.length()){
            return false;
        }
        int[] counter = new int[26];
        for (int i = 0; i < s.length(); i++){
            counter[s.charAt(i) - 'a'] += 1;
            counter[t.charAt(i) - 'a'] -= 1;
        }
        for (int i = 0; i < counter.length; i++){
            if (counter[i] > 0){
                return false;
            }
        }
        return true;

        
    }
}
